cinder/volume/manager.py

```diff
@@ -1178,7 +1178,7 @@ def revert_to_snapshot(self, context, volume, snapshot):
         self._notify_about_snapshot_usage(context, snapshot, "revert.end")
 
     @objects.Snapshot.set_workers
-    def create_snapshot(self, context, snapshot):
+    def create_snapshot(self, context, snapshot, backend=None):
```

Comment:
This is a bit weird, since they're unused. I'd prefer to not set a default (`=None` doesn't hint at them being unused, but rather signals to me that care was taken to give it a sane default, for whatever reason).
AFAICS there's no commonly accepted convention for unused arguments: https://stackoverflow.com/questions/10025680/how-can-i-denote-unused-function-arguments

I think
```suggestion
    def create_snapshot(self, context, snapshot, _backend):
```
would probably be my favorite, although it's got its detractors in the SO discussions linked above.

Reply:
good catch, this was a remnant of my first approach to doing this, which I didn't like.  removing.